sql/core/src/test/scala/org/apache/spark/sql/GeneratorFunctionSuite.scala

```diff
@@ -344,12 +344,35 @@ class GeneratorFunctionSuite extends QueryTest with SharedSparkSession {
     }
   }
 
+  test("Supported nested inner generators") {
```

Comment:
Can we add more generators? Or, shall we mention `explode/explode_outer`?

Reply:
Yea, I'll try.